tests/languages/json-source-code.test.js

```diff
@@ -14,7 +14,7 @@ import dedent from "dedent";
 
 //-----------------------------------------------------------------------------
 // Tests
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
```

Comment:
```suggestion
//-----------------------------------------------------------------------------
```

I think this line was modified by mistake.